### PR TITLE
feat: add background to game

### PR DESCRIPTION
**Pull request by JheremyTancara**
# Pull Request

## Type of Change

- [x] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality
      to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: src/utils/assets/elements/touchable/coin.py [hunk 1]
```diff
@@ -1,9 +1,20 @@
...
+
+COIN_3 = transform.scale(
+    image.load(COIN_DIR + "coin_3.png"), GENERAL_SIZE
+)
 
-COIN_1 = create_image(COIN_DIR + "coin_1.png", COIN_SIZE)
-COIN_2 = create_image(COIN_DIR + "coin_2.png", COIN_SIZE)
-COIN_3 = create_image(COIN_DIR + "coin_3.png", COIN_SIZE)
-COIN_4 = create_image(COIN_DIR + "coin_4.png", COIN_SIZE)
+COIN_4 = transform.scale(
```
Comment (denis-gandel) [inline src/utils/assets/elements/touchable/coin.py]: You are making unnecessary changes, for example you are changing the size of the coins, since by giving them the size of 60 x 60 pixels, you will change their shape and it will be super ugly.

File: src/utils/constants.py [hunk 2]
```diff
@@ -36,11 +36,8 @@
 TO_SECONDS = 1000
 
 # GENERAL SIZES
-# 8 x 8 pixels
-SMALL_SIZE = (30, 30)
-
 # 16 x 16 pixels
-GENERAL_SIZE = (60, 60)
+GENERAL_SIZE = (50, 50)
```
Comment (denis-gandel) [inline src/utils/constants.py]: The idea is to keep 60 x 60, because this is the designated size for the screen, since 15 blocks high must fit on the screen in order to maintain the positions of the elements.

File: src/utils/assets/elements/touchable/coin.py (hunk 1 above)
Comment (JheremyTancara) [reply src/utils/assets/elements/touchable/coin.py]: The size of the object is centered inside each box of the screen where the size is 50x50 and by doing it that way the objects are not properly accommodated.

File: src/utils/constants.py (hunk 2 above)
Comment (JheremyTancara) [reply src/utils/constants.py]: The JSON configuration work was done previously before the sizes, doing it again is double effort.

Comment (JheremyTancara) [reply src/utils/constants.py]: All boxes within the JSON were set to 50x50

File: src/utils/directories.py [hunk 3]
```diff
@@ -32,22 +31,34 @@
...
+MOUNTAIN_BACKGROUND_DIR = BACKGROUND_DIR + "mountain/"
+TREE_BACKGROUND_DIR = BACKGROUND_DIR + "tree/"
 
 # == Blocks ==
 # == blocks name ==
 BRICK_DIR = BLOCKS_DIR + "brick/"
 BROKE_BLOCK_DIR = BLOCKS_DIR + "broke_block/"
 COIN_DIR = BLOCKS_DIR + "coin/"
-MASTERY_BLOCK_DIR = BLOCKS_DIR + "mastery_block/"
+MYSTERY_BLOCK_DIR = BLOCKS_DIR + "mistery_block/"
```
Comment (denis-gandel) [inline src/utils/directories.py:61]: Unnecessary change

File: src/utils/directories.py [hunk 4]
```diff
@@ -126,26 +137,38 @@
 CUMPA_LVL_1_BRAKE_DIR = CUMPA_LVL_1_DIR + BRAKE_DIR
 CUMPA_LVL_1_DIED_DIR = CUMPA_LVL_1_DIR + DIED_DIR
 CUMPA_LVL_1_DOWN_DIR = CUMPA_LVL_1_DIR + DOWN_DIR
+CUMPA_LVL_1_DOWN_FRONT_DIR = CUMPA_LVL_1_DOWN_DIR + FRONT_DIR
```
Comment (denis-gandel) [inline src/utils/directories.py:141]: You are reintroducing changes that were already eliminated, due to their unnecessary nature.

File: src/utils/surfaces.py [hunk 5]
```diff
@@ -4,56 +4,79 @@
...
-    MASTERY_BLOCK_2,
-    PIPE_BODY_STYLE_1,
-    PIPE_BODY_STYLE_2,
-    PIPE_BODY_STYLE_3,
-    PIPE_BODY_STYLE_4,
+    WALL_BLOCK,
+    MYSTERY_BLOCK_1,
+    MYSTERY_BLOCK_2,
     PIPE_HEAD,
+    PIPE_BODY,
```
Comment (denis-gandel) [inline src/utils/surfaces.py]: The idea is that the pipes have different styles to be able to give it different aspects and not be so equal.

File: src/utils/directories.py (hunk 3 above)
Comment (JheremyTancara) [reply src/utils/directories.py:61]: Mastery block in Spanish is 'Bloqueo de dominio' only changed to a more representative name

File: src/utils/directories.py (hunk 4 above)
Comment (JheremyTancara) [reply src/utils/directories.py:141]: It was a rebasing error because when the feature was uploaded it was rebased 3 times, and just one of them modified quite a few folders made by this PR.

File: src/utils/surfaces.py (hunk 5 above)
Comment (JheremyTancara) [reply src/utils/surfaces.py]: It will be fixed, before uploading the feature

**Comment by denis-gandel** [comment]
In this case in our game, the MAStery block is positioned 2 blocks above the first nearby platform, but in the real game it is a difference of 3 blocks. The problem would be at the beginning of the map. 

SUPER PITON BROS:
acb6bd38-4a0e-42f3-95ce-4cb6671f2b9b (2)

ORIGINAL SUPER MARIO BROS:
acb6bd38-4a0e-42f3-95ce-4cb6671f2b9b

**Review by MorveN11 — APPROVED** [review]
@JheremyTancara good job
